src/services/cheWorkspaceClient/index.ts

```diff
@@ -32,7 +31,6 @@ export class CheWorkspaceClient {
    */
   constructor() {
     this.baseUrl = '/api';
-    this.websocketContext = '/api/websocket';
 
     this.originLocation = new URL(window.location.href).origin;
 
```

Comment:
@JPinkney You have fixed https://github.com/eclipse/che/issues/18311  and one of the described solutions was:
'che-workspace-client should be able to expose an Axios instance'.
You did not implement it. So what do you propose to do with this temporary code 
``` this.axios = (WorkspaceClient as any).createAxiosInstance({ loggingEnabled: false });  ```?

Reply:
I think we can keep it for now, Serhii opened up another issue to cover that: https://github.com/eclipse/che/issues/18803

Reply:
OK